### ISSUE TITLE
Feature request: open and manage Chrome extensions

### ISSUE THREAD

**Issue by nikhilmaddirala** [dbuezas/chrome-palette#4]
I have several Chrome extensions and the only way to open or manage is to select "Open Extensions" in Chrome Palette and then select the specific extension I want. Instead it would be great if I could directly type the name of the extension into Chrome Palette to open or manage it. Thanks in advance for considering this suggestion! 😊

**Comment by dbuezas** [comment]
It looks like it should be possible https://developer.chrome.com/docs/extensions/reference/api/management
I'll look into it

**Comment by dbuezas** [comment]
<img width="612" alt="image" src="https://github.com/user-attachments/assets/c4218085-ec8a-4aa2-848a-a397394ac1eb">

V2.1.0 is being reviewed by google and has this feature

**Comment by nikhilmaddirala** [comment]
That's amazing — thank you for being the most responsive Chrome extension developer I've ever met! 😊

I'm also curious to know if you have any other future improvement plans / roadmap for this extension?

**Comment by dbuezas** [comment]
No roadmap, I just add things when I miss them :)

**Comment by dbuezas** [comment]
One thing I do miss is a command to resize the window to the left 50% of the screen and then put the current tab in a new window on the right 50% of the screen

**Comment by nikhilmaddirala** [comment]
> One thing I do miss is a command to resize the window to the left 50% of the screen and then put the current tab in a new window on the right 50% of the screen

What operating system do you use? On Macos this could easily be accomplished using something like Alfred / BetterTouchTool / etc.

**Comment by dbuezas** [comment]
Yes I use better touch tool for that, but the extension could do it in one step

**Comment by nikhilmaddirala** [comment]
Alfred definitely supports multi-step actions like this with a single command. I think BetterTouchTool can also do this.

**Comment by dbuezas** [comment]
FYI: extensions list is live!
Also split screen in review (should be out tomorrow)

**Comment by nikhilmaddirala** [comment]
Hi @dbuezas - There seems to be some regression and the extensions list is no longer working for me (but split screen is working).

**Comment by dbuezas** [comment]
Are you on v2.2.2.0?
It does work for me.
Please try this:
* Right click on the extension pop-up
* Inspect
* See if there are any errors in the console
* Enter search extensions section
* Check for errors again
* Paste any error here

**Comment by nikhilmaddirala** [comment]
Yes I'm on v2.2.2.0, and there are no console errors, but unfortunately
it's not working - e.g. I type the name of a known extension and there are
no results.

**Comment by dbuezas** [comment]
You have to get Into "search extensions" first

**Comment by dbuezas** [comment]
Or type `e>` first

**Comment by nikhilmaddirala** [comment]
Oops - this was just user error, my bad!

**Comment by dbuezas** [comment]
I could put everything at the root, but with history, tabs and bookmarks it gets crazy big. Also I opted to keep the root list fixed so you always know you'll get the same action with the same input
